client/opentracing/src/main/java/io/opentracing/APMSpanBuilder.java

```diff
@@ -34,20 +34,24 @@
 
     private TraceRecorder recorder;
     private ContextSampler sampler;
+    private String serviceName;
+    private String buildStamp;
 
     /**
      * @param operationName The operation name
      * @param recorder The trace recorder
      */
-    APMSpanBuilder(String operationName, TraceRecorder recorder, ContextSampler sampler) {
+    APMSpanBuilder(String operationName, TraceRecorder recorder, ContextSampler sampler, String serviceName, String buildStamp) {
```

Comment:
Another alternative is the `Sampler`. The `TracerRecorder` sounded more like the right place to me, but if you think the `Sampler` would be more appropriate, then `Sampler` it is :)